.github/renovate.json

```diff
@@ -1,5 +1,11 @@
 {
-  "extends": ["github>balena-os/renovate-config"],
+  "extends": [
+    "github>balena-os/renovate-config",
+    ":automergeStableNonMajor",
+    ":automergePatch",
+    ":automergeDigest",
+    ":automergeTypes"
+  ],
```

Comment:
In addition to this, we need `automerge: false` in the root to override the inherited balena-os setting.